tests/unit/checkboxradio/checkboxradio_common.js

```diff
@@ -0,0 +1,15 @@
+TestHelpers.commonWidgetTests( "checkboxradio", {
+	noDefaultElement: true,
```

Comment:
The default element should have been removed. It's not possible for it to work because it needs a label.